book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md

```diff
@@ -37,13 +31,13 @@ in case it is not, then set it
 
 Now, run the docker container `kmeans` on your duckiebot
 
-    laptop $ docker -H ![ROBOT_NAME].local run -it --memory="800m" --memory-swap="1.8g" --net host --privileged -v /data:/data --name kmeans manishpra/devel-kmeans-unsup:master18 /bin/bash
+    laptop $ docker -H ![ROBOT_NAME].local run -it --memory="800m" --memory-swap="1.8g" --net host --privileged -v /data:/data --name kmeans duckietown/devel-kmeans-unsup:master18 /bin/bash
 
 **Step 2**: Launch the camera_kmeans launch file
 
     duckiebot $ roslaunch pi_camera camera_kmean.launch veh:=![ROBOT_NAME] k:=![k] reduction:=![reduction]
```

Comment:
duckiebot-container ?

Reply:
It is duckietown container. That's correct.

Reply:
sorry  I meant you can write "duckietbot-container $" in place of "duckiebot $"